### PR TITLE
Add skywater adk node

### PR DESCRIPTION
**Pull request by alexcarsello**
Adds a skywater 130 adk that pull the standard view from our new sky130 adk repo.

### CODE REVIEW

File: adks/skywater-130nm/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+README
+==========================================================================
+Author : Alex Carsello
+Date   : July 12, 2022
+
+This ASIC design kit uses Skywater 130nm.
```
Comment (ctorng) [inline adks/skywater-130nm/README.md:6]: Phrasing should probably be "This is an ASIC design kit for Skywater 130nm"

File: adks/skywater-130nm/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+More information is available in the README of the standard base kit
+located here:
+
+- https://github.com/mflowgen/skywater-130nm
```
Comment (ctorng) [inline adks/skywater-130nm/README.md:11]: Can we mention here what we discussed about how we will handle versioning for now? The repo is currently set up as a set of pre-built versions of the ADK. This node will pull the corresponding version of the ADK's standard view.

Maybe a "version" parameter which defaults to "v0.0.0" for now?

**Review by ctorng — CHANGES_REQUESTED** [review]

File: adks/skywater-130nm/README.md (hunk 2 above)
Comment (alexcarsello) [reply adks/skywater-130nm/README.md:11]: yes that's what I'm doing

Comment (alexcarsello) [reply adks/skywater-130nm/README.md:11]: Done. For some reason the folder in the downloaded zip file is called skywater-130nm-0.0.0 and not skywater-130nm-v0.0.0

Comment (ctorng) [reply adks/skywater-130nm/README.md:11]: Great! Can we just put what I said into a comment at the top of this configure.yml? and then I think we are done here.

**Pull request merged**